### ISSUE TITLE
Auth state is an invalid state-After I delete amplify account in Kotlin and back to SiginScreen

### ISSUE THREAD

**Issue by kylequang** [aws-amplify/amplify-android#2288]
``InvalidStateException{message=Auth` state is an invalid state, cannot process the request., cause=null, recoverySuggestion=Operation performed is not a valid operation for the current auth state.}`

![Screenshot 2023-02-15 at 18 44 29](https://user-images.githubusercontent.com/67398186/219018603-ecc344b5-a284-4f87-bcc7-4b6bd09a9e7d.png)

I deleted amplify account by `Amplify.Auth.deleteUser()`
After that I back to SignInScreen and I use `Amplify.Auth.fetchAuthSession()` to check isSignedIn but I received error above. It still login -> isSignedIn return true

Anyone help me ???

**Comment by tjleing** [comment]
Hey @kylequang, thanks for reaching out!  Can you specify which version of the library you're using (found in `build.gradle`)?

**Comment by kylequang** [comment]
I use : `implementation 'com.amplifyframework:aws-auth-cognito:2.2.1'`

**Comment by kylequang** [comment]
```
private suspend fun deleteAccountAmplify(): Boolean {
        return try {
            Amplify.Auth.deleteUser()
            Log.i("AuthQuickStart", "Delete user succeeded")
            true
        } catch (error: AuthException) {
            Log.e("AuthQuickStart", "Delete user failed with error", error)
            false
        }
    }
```

After delete account success. I back to other screen (ex: signIn). I use bellow code to check isSignIn

**Comment by kylequang** [comment]
@sdhuka sd
@tjleing 
@div5yesh div
Is this a bug in your library? let me know if i'm wrong

**Comment by gpanshu** [comment]
Hi @kylequang this is indeed a bug in our library. I have created a PR for this fix. Thank you so much for reporting it. I will advise once this is released.

**Comment by tjleing** [comment]
@kylequang we have released Amplify v2.3.0 which contains @gpanshu's fix.  Please let us know if that resolves your issue!
